en/resources/books-blogs.md

```diff
@@ -6,7 +6,7 @@ lang: en
 redirect_from: "/resources/books-blogs.html"
 ---
 
-# Books and blogs
+# Books, blogs and video tutorials
```

Comment:
This is a real nitpick, but we generally use the serial or Oxford comma... So:
> Books, blogs, and video tutorials

Just make this small change.

Reply:
Sure, I changed it. Thanks for the prompt reply.